plugins/Toolbox/src/CloudSync/SubscribedPackagesModel.py

```diff
@@ -37,27 +37,22 @@ def hasIncompatiblePackages(self) -> bool:
                 return True
         return False
 
-        # Sets the "is_compatible" to True for the given package, in memory
```

Comment:
This is probably a forgotten comment line which doesn't apply to the function beneath it.